src/main/java/dev/jbang/source/buildsteps/NativeBuildStep.java

```diff
@@ -31,6 +31,12 @@ public Project build() throws IOException {
 		List<String> optionList = new ArrayList<>();
 		optionList.add(resolveInGraalVMHome("native-image", project.getJavaVersion()));
 
+		final int actualVersion = JavaUtil.javaVersion(project.getJavaVersion());
+		if (actualVersion == 19) {
+			optionList.add("--enable-preview");
+			optionList.add("--add-modules=jdk.incubator.concurrent");
```

Comment:
Why is this option necessary exactly?

Reply:
Same question here - why would we enable this by default for kotlin? We dont enable it by default in java

Reply:
Sorry thought this was in kotlin part, most definitely Should not be actively enabled for native by default. Need to add explicit support for it to enable for native.

Reply:
> Since virtual threads are a preview feature in JDK 19, they are also a preview feature for GraalVM Native Image. They can be enabled with the new option --enable-preview.

https://docs.oracle.com/en/graalvm/enterprise/22/docs/release-notes/

Reply:
Ok, understood. In that case we really need https://github.com/jbangdev/jbang/issues/1493 instead of adding a bunch of hard-coded options.

Reply:
Created PR #1494 for #1493

Reply:
@linux-china can you redo this PR without the enable preview /add modules flags?